scripts/2.TCGA-MLexample-covariates.py

```diff
@@ -349,7 +354,7 @@ def get_coefficients(classifier, X_mat):
 ax = sns.distplot(predict_df.query("status == 1").decision_function, hist=False, label='Positives')
 
 
-# In[27]:
+# In[26]:
 
 ax = sns.distplot(predict_df.query("status == 0").probability, hist=False, label='Negatives')
 ax = sns.distplot(predict_df.query("status == 1").probability, hist=False, label='Positives')
```

Comment:
> I'm not a fan of the decision_function plot, but the probability one is more interpretable.

Yep we had the decision_function plot for back when some classifiers didn't give probabilities. Let's remove it and keep just the probability one